application/frontend/controllers/UserController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+namespace frontend\controllers;
+
+use common\models\User;
+use frontend\components\BaseRestController;
+use Yii;
+use yii\web\NotFoundHttpException;
+
+class UserController extends BaseRestController
+{
+    public function actionCreate(): User
+    {
+        $existingUser = User::findOne([
+            'employee_id' => Yii::$app->request->getBodyParam('employee_id')
+        ]);
+
+        $user = $existingUser ?? new User();
+
+        $user->attributes = Yii::$app->request->getBodyParams();
```

Comment:
It's totally reasonable, I completely understand where you're coming from.  It was a little too "magicky" for me when I first saw it but I came around to liking the idea, you know how some designs grow on you once you've begun using and appreciate where the authors were coming from?  FWIW, there is still a whitelisting process, it's just be done via scenarios instead of through getters in the controller.   This aversion you're having may simply be about the notion of who's the main actor here, the controller or the model...let's see how it strikes @fillup , if the style is a no-go for you all because it's too different from what you want to see, I'll refactor.